resources/assets/js/components/ErrorToast.tsx

```diff
@@ -1,94 +1,57 @@
-import React, { useEffect } from "react";
-import { connect } from "react-redux";
-import {
-  injectIntl,
-  FormattedMessage,
-  WrappedComponentProps,
-} from "react-intl";
-import { ApiError } from "redux-api-middleware";
-import { ErrorEntity } from "../store/Error/errorReducer";
-import { clearErrors } from "../store/Error/errorActions";
-import { getRecentError } from "../store/Error/errorSelector";
-import { RootState } from "../store/store";
-import { DispatchType } from "../configureStore";
+import React, { useCallback, useContext, useEffect, useRef } from "react";
+import { defineMessages, useIntl } from "react-intl";
+import { ErrorContext } from "./ErrorContainer";
+import Alert from "./H2Components/Alert";
 
-const constructErrorMessage = (error: ErrorEntity): string => {
-  if (error.error instanceof ApiError) {
-    // if error was of type redux-api-middleware/ApiError, we may be able to extract extra info from the response.
-    const apiError = error.error;
-    if (apiError.status && apiError.response && apiError.response.message) {
-      return `${apiError.status} - ${apiError.response.message}`;
-    }
-  }
-  return error.message;
-};
-interface ErrorToastProps {
-  error: ErrorEntity;
-  dispatchClearErrors: () => void;
-}
-const ErrorToast: React.FC<ErrorToastProps & WrappedComponentProps> = ({
-  error,
-  dispatchClearErrors,
-}): React.ReactElement => {
-  useEffect((): (() => void) => {
-    const timer = setInterval((): void => {
-      dispatchClearErrors();
-      clearInterval(timer);
-    }, 3500);
-    return (): void => {
-      clearInterval(timer);
-    };
-  }, [dispatchClearErrors, error]);
+const TOAST_SELF_DISMISS_TIMER = 3500;
```

Comment:
`TOAST_SELF_DISMISS_TIMER` isn't being used anywhere. Same with `useEffect` and `useRef` in the imports section.